core/src/main/java/org/jahia/modules/external/ExternalContentStoreProviderFactoryImpl.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2002-2022 Jahia Solutions Group SA. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jahia.modules.external;
+
+import org.jahia.services.content.JCRSessionFactory;
+import org.jahia.services.content.JCRStoreService;
+import org.jahia.services.sites.JahiaSitesService;
+import org.jahia.services.usermanager.JahiaGroupManagerService;
+import org.jahia.services.usermanager.JahiaUserManagerService;
+import org.osgi.service.component.annotations.Reference;
+
+public class ExternalContentStoreProviderFactoryImpl implements ExternalContentStoreProviderFactory {
+
+    private JahiaUserManagerService userManagerService;
+    private JahiaGroupManagerService groupManagerService;
+    private JahiaSitesService sitesService;
+    private JCRStoreService jcrStoreService;
+    private JCRSessionFactory sessionFactory;
+
+    private ExternalProviderInitializerService externalProviderInitializerService;
+
+    @Reference
```

Comment:
I think the @Reference annonations are not required since it's not using DS to register the service but Spring.

Reply:
Yes you are right, I guess I put those out of habit :) I'll remove them.